src/graphql/queries/events.queries.ts

```diff
@@ -1,11 +1,38 @@
 import { gql } from "@apollo/client";
 
+export interface FoodDrive {
+  id: string;
+  isOpen: boolean;
+  name: string;
+  startDate: string;
+  endDate: string;
+  location: {
+    address: string;
+    coords: {
+      longitude: string;
+      latitude: string;
+    };
+  };
+}
+export interface GetFoodDrives {
+  getFoodDrives: FoodDrive[];
+}
+
 export const GET_FOOD_DRIVES = gql`
   query Query {
     getFoodDrives {
       id
+      isOpen
       name
-      description
```

Comment:
Are we not displaying the description for the food drive?

Reply:
Not in the Event list